### ISSUE TITLE
Configure for access using another OneDrive account

### ISSUE THREAD

**Issue by GithSTeve** [OneDrive/samples#79]
I am new to the site 
So please forgive me If I am not conveying this in the correct forum.

Using File Picker V 7.2 
I have reviewed the pages for 7/8 configuration at least ten times.

Currently in the application I am listed as the owner. I registered  it .

So the question Is there a way  (In version 7 or 8) to configure my application to display and allow users access to files and folders from their One Drive directory ?

Just so I am clear The configuration requires the client ID, but that is for the application. Is there a way to configure so the client goes to a page, to access and work with files and folders that belongs to their OneDrive directory and not the directory of the application. 

In other words, the app is under an Azure account (that does not  have a one drive account) and I am trying to configure so the client is using the app to access files and folders in their OneDrive account via my  app which is configured to allow it. 

And currently its not working 

I hope I am making sense.

**Comment by Pasatedeverga** [comment]
[URL_1715188573018.csv](https://github.com/user-attachments/files/16971308/URL_1715188573018.csv)

**Comment by JCrew0** [comment]
@GithSTeve Yes, the purpose of the OneDrive File Picker and File Browser is to allow users to log into their OneDrive account on  your application and select files from their OneDrive. I'm unclear what you mean by "directory of the application", can you elaborate a bit more on what exactly is going wrong right now?
